### ISSUE TITLE
[WAC] - Page fails to load

### ISSUE THREAD

**Issue by PeterRoots** [MicrosoftDocs/Windows-Admin-Center-Ideas-and-Feedback#76]
**Gateway Version**: 1.3.2111.01001

**To Reproduce**
while using WAC periodically I find when I change page, eg refresh a page that seems to have frozen or try to go back to server manager from a server WAC fails to load
      Bad Request - Request Too Long
      HTTP Error 400. The size of the request headers is too long.
I have tried reloading/refreshing the page, browsing to the base URL, closing the tab in my browser and relaunching WAC, all to no effect.
Restarting the service on the gateway server is the only fix I have found so far although if I wait long enough (5 mins perhaps) then it may start to work again on its own

**Comment by trungtran-msft** [comment]
Hey @PeterRoots, 

Can you please provide a .HAR file or anything you can so we can reproduce this issue on our side?

**Comment by PeterRoots** [comment]
@trungtran-msft where do I find/how do I produce a .HAR file?

**Comment by trungtran-msft** [comment]
@PeterRoots 

Please take a look at this article: https://support.zendesk.com/hc/en-us/articles/4408828867098-Generating-a-HAR-file-for-troubleshooting

**Comment by PeterRoots** [comment]
This is rather random so difficult to catch but has just happened again so .har attached, well it is attached as a zip file as you dont apparently support har files for upload
[amr-mon-01.zip](https://github.com/MicrosoftDocs/Windows-Admin-Center-Ideas-and-Feedback/files/7641264/amr-mon-01.zip)

**Comment by trungtran-msft** [comment]
Thank you @PeterRoots, I've filed a bug for our engineers to investigate.

**Comment by trungtran-msft** [comment]
Hey @PeterRoots, are you using different domains?

**Comment by PeterRoots** [comment]
yes @trungtran-msft  but as WAC no longer remembers the log on username for individual servers we have to have 2 separate gateways, one for each domain. I see this issue intermittently on both gateways but recently the frequency seems to have reduced somewhat

**Comment by davannaw-msft** [comment]
We've made a lot of improvements in Windows Admin Center since this issue was raised. Please the most recent version a try and let us know if the issue persists.
